crew: Add a new command for showing disk statistics

supechicken opened the pull request:
This PR adds a new command for showing disk statistics, which can be useful for users who:
- Want to free up disk space by removing packages due to insufficient storage
- Want to know what the disk space is actually used for

The new command will be `crew diskstat`. By default, the package size and its approximate percentage (which can be found in the color bar) for the 12 largest installed packages will be shown.

<img width="500" height="719" alt="Screenshot 2025-08-22 23 56 03" src="https://github.com/user-attachments/assets/feea4933-d3a9-4eb6-988d-fd181f43cda4" />

### Run the following to get this pull request's changes locally for testing.
```bash
CREW_REPO=https://github.com/supechicken/chromebrew.git CREW_BRANCH=crew_new_command crew update
```

uberhacker commented on commands/diskstat.rb:
```diff
@@ -0,0 +1,86 @@
+    total_size     = size_of_all_packages.sum(&:last).to_f # Total size of all installed packages
+    other_size     = total_size                            # Total size of all installed packages, excluding the top 12 one
+    bar_length     = 0
+    bar_components = size_of_all_packages[..11].map.with_index do |(_, size), i|
```
What are you thoughts about making the 11 configurable with a switch option and/or environment variable?

supechicken replied:
All packages will be printed below, but only the top 12 are inside the bar chart.

EDIT: Just found a way to display up to 24 different colors

uberhacker commented on commands/help.rb:
```diff
@@ -34,6 +34,12 @@ def self.help(command)
+        Show statistics about the disk space occupied by installed Chromebrew packages.
+        Usage: crew diskstat [-a|--all]
+        Only 12 of the largest packages will be shown by default. Use `--all` to show the occupied disk space for all installed packages.
+      EOT
```
Also add to README.md?

supechicken commented on bin/crew:
```diff
@@ -15,23 +15,6 @@ require 'mkmf'
-require_relative '../commands/remove'
-require_relative '../commands/search'
-require_relative '../commands/sysinfo'
-require_relative '../commands/whatprovides'
```
I have simplified the loading logic into a `Dir.glob` loop below

supechicken commented:
Here is the final result with suggestions applied, with an adjustable package count:
<img width="500" height="743" alt="Screenshot 2025-08-23 04 23 42" src="https://github.com/user-attachments/assets/0c72c078-e937-49aa-8ea7-41c06a5c8c84" />

(I have also added a count for userdata directories like `~/.config` and `~/.cache`)

> This is a great idea @supechicken! I added a few comments. Should we also add a test for this command?

But I have no idea how to make a test case for this😅 And it is impossible to check if this is working correctly by simple tests.

supechicken commented:
The size count doesn't look right when a package count is specific, will look into it tomorrow

Zopolis4 reviewed (approved):
nice simplifications, thank you for splitting out the command from the start, good work overall!
